MIDAS/src/main.cpp

```diff
@@ -5,6 +5,15 @@
  */
 RocketSystems systems;
 
+#ifdef HILSIM
+#include "hilsim/hilsim.pb.h"
```

Comment:
remove this hilsim stuff for now it's not necessary for this pr

Reply:
This is my fault for trying to work on 3 tickets at the same time